src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -123,13 +104,50 @@ public string Description
             get { return Model.Description; }
         }
 
+        public string Class
+        {
+            get
+            {
+                return Model.Categories.Count > 1 ? Model.Categories.Last() : String.Empty;
+            }
+        }
+
+        public string Category
+        {
+            get
+            {
+                return Model.Categories.First();
+            }
+        }
+
+        public SearchElementGroup Group
+        {
+            get { return Model.Group; }
+        }
+
+        public string GroupIconName
+        {
```

Comment:
we should build a dictionary of items and use it here. Say the dictionary as FontAwesomeDict with key as Create , and value as Icon name. Instead of switch you can directly refer as FontAwesomeDict [Model.Group]. So, even if we have new categories, we can just add to FontAwesomeDict. of course. check if the dict.containskey.